src/pc-fan-element.stories.ts

```diff
@@ -0,0 +1,40 @@
+import { html } from 'lit';
+import { forceReRender } from '@storybook/web-components';
+import './pc-fan-element';
+
+class FanAnimation {
+  angle = 0;
+
+  step() {
+    this.angle = this.angle + 5;
+
+    if (this.angle >= 360) {
+      this.angle = 0;
+    }
+  }
+}
+
+const fanAnimation = new FanAnimation();
+setInterval(() => {
+  fanAnimation.step();
+  forceReRender();
+}, 10);
+
+export default {
+  title: 'PC Fan',
+  component: 'wokwi-pc-fan',
+  argTypes: {
+    angle: { control: { type: 'number', min: 0, max: 360, step: 1 } },
```

Comment:
How about using a "range" type for the control? I think it'll be more user friendly